RevenueCatUI/CustomerCenter/Data/SubscriptionInformation.swift

```diff
@@ -66,29 +60,48 @@ struct SubscriptionInformation {
         self.price = price
         self.expirationOrRenewal = expirationOrRenewal
         self.productIdentifier = productIdentifier
-        self.willRenew = willRenew
-        self.active = active
         self.store = store
     }
 
     init(entitlement: EntitlementInfo,
          subscribedProduct: StoreProduct? = nil,
          dateFormatter: DateFormatter = DateFormatter()) {
-        // swiftlint:disable:next todo
-        // TODO: support non-consumables
         dateFormatter.dateStyle = .medium
 
         self.title = subscribedProduct?.localizedTitle
         self.explanation = entitlement.explanation
         self.durationTitle = subscribedProduct?.subscriptionPeriod?.durationTitle
         self.price = entitlement.priceBestEffort(product: subscribedProduct)
         self.expirationOrRenewal = entitlement.expirationOrRenewal(dateFormatter: dateFormatter)
-        self.willRenew = entitlement.willRenew
         self.productIdentifier = entitlement.productIdentifier
-        self.active = entitlement.isActive
         self.store = entitlement.store
     }
 
+    init(product: StoreProduct,
+         expirationDate: Date?,
+         dateFormatter: DateFormatter = DateFormatter()) {
+        // We don't have enough information to determine if the subscription will renew or not because we
+        // are loading the information from the product without entitlement information.
+        // We also assume that the subscription is active.
+        // We also assume that the subscription will renew the earliest possible renewal date and this is the
+        // product with the earliest renewal date.
```

Comment:
We have to add the Subscription information to the CustomerInfo. It's already being returned in the JSON, we just don't have it in the `CustomerInfo` class. 

Once we have that, we can fix this.